### PR TITLE
Design edits

### PR DESCRIPTION
**Pull request by mukeshchugani10**
This PR fixes multiple issues:

1. Spacing and alignment of elements in the hero part of homepage as suggested by @shubhamk008.
2. Margins of other elements in the homepage.
3. Favicon with transparent bg.
4. Adds style for active page on navbar (#16 )
5. Most of the tasks in #46 .

### CODE REVIEW

File: src/pages/contact.tsx [hunk 1]
```diff
@@ -84,27 +84,28 @@ const Contact: NextPage = () => {
...
               </div>
             </div>
           </div>
           <div className="flex mb-10 mt-10">
             <Link
               className="mr-10"
-              href="https://github.com/healthyregions"
-              title="@heathyregions on GitHub"
+              href="https://github.com/healthyregions/SDOHPlace"
+              title="@sdohplace on GitHub"
```
Comment (mradamcox) [inline src/pages/contact.tsx:102]: This should still reference @HealthyRegions in the title, or something like "View code on GitHub".  There isn't an @sdohplace user on github.

File: src/components/homepage/footer/footer.tsx [hunk 2]
```diff
@@ -185,16 +185,12 @@ const Footer = (): JSX.Element => {
 
             <div className="flex flex-row gap-5">
               <a
-                href="https://github.com/healthyregions"
-                title="@heathyregions on GitHub"
+                href="https://github.com/healthyregions/SDOHPlace"
+                title="@sdohplace on GitHub"
```
Comment (mradamcox) [inline src/components/homepage/footer/footer.tsx:189]: See other comment.

**Review by mradamcox — COMMENTED** [review]
Just one small change request to a link title, otherwise looks good.

File: src/pages/contact.tsx (hunk 1 above)
Comment (mukeshchugani10) [reply src/pages/contact.tsx]: Fixed in [4f300b0](https://github.com/healthyregions/SDOHPlace/pull/50/commits/4f300b0bf06615fa1e1144a4b49d8929cb11b591)

File: src/components/homepage/footer/footer.tsx (hunk 2 above)
Comment (mukeshchugani10) [reply src/components/homepage/footer/footer.tsx]: Fixed in [4f300b0](https://github.com/healthyregions/SDOHPlace/pull/50/commits/4f300b0bf06615fa1e1144a4b49d8929cb11b591)

**Pull request merged**